### PR TITLE
Fixing build for MSVC.

### PR DESCRIPTION
**Pull request by Predelnik**
Since I don't know what exactly is wrong I'm playing a bit of guessing game here. But I've tried to apply your suggestions.

Maybe if it doesn't work I need to learn how to test things locally.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1697,8 +1691,6 @@ install(TARGETS ${_INSTALL_LIBS}
 ##### Export files #####
 if (APPLE)
   set(PKG_PREFIX "SDL2.framework/Resources")
```
Comment (ruslo) [inline CMakeLists.txt]: I guess this line should be fixed too. `lib/cmake/SDL2` will work for all cases.

Comment (ruslo) [reply CMakeLists.txt]: > ${CMAKE_CURRENT_SOURCE_DIR}/SDL2Config.cmake

This should be removed. Config generated from `cmake/Config.cmake.in`.

**Review by ruslo — CHANGES_REQUESTED** [review]
> Maybe if it doesn't work I need to learn how to test things locally

Testing locally described here:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#testing-locally

Note that in current workflow you should test new version of package remotely youself before submitting it to main repository:
* https://docs.hunter.sh/en/latest/creating-new/update.html#testing

Meaning that situation when some toolchains failed like in https://github.com/ruslo/hunter/pull/1270 should not happen.

**Comment by Predelnik** [comment]
So I've managed to run CI on my fork. After proposed changes, looks good:
https://ci.appveyor.com/project/Predelnik/hunter/build/1.0.2
https://travis-ci.org/Predelnik/hunter

**Review by ruslo — APPROVED** [review]

**Comment by ruslo** [comment]
Released: https://github.com/hunter-packages/SDL2/releases/tag/v2.0.7-p1

**Pull request merged**